Index: database/database.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import os.path\r\n\r\nimport psycopg\r\nfrom flask import current_app\r\n\r\n\r\ndef create_database():\r\n    # load schema.sql file into variable\r\n    schema: str\r\n    with open(os.path.join(os.path.abspath(\".\"), \"database\", \"schema.sql\"), \"r\") as file:\r\n        schema = file.read()\r\n\r\n    # Connect to an existing database\r\n    with psycopg.connect(\"host=localhost user=postgres password=Prktsql1. port=5432 dbname=postgres\") as conn:\r\n        # Open a cursor to perform database operations\r\n        with conn.cursor() as cur:\r\n            # Execute the schema script to initialize the database\r\n            cur.execute(schema)\r\n            # Commit the changes to the database\r\n            # When this is not called, no changes will be made to database.\r\n            conn.commit()
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/database/database.py b/database/database.py
--- a/database/database.py	(revision 0e593b46ba452056e56fe87265aa3ec40e4dfe34)
+++ b/database/database.py	(date 1731351986630)
@@ -11,7 +11,7 @@
         schema = file.read()
 
     # Connect to an existing database
-    with psycopg.connect("host=localhost user=postgres password=Prktsql1. port=5432 dbname=postgres") as conn:
+    with psycopg.connect("host=localhost user=postgres password=progre port=5432 dbname=postgres") as conn:
         # Open a cursor to perform database operations
         with conn.cursor() as cur:
             # Execute the schema script to initialize the database
Index: database/user.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import psycopg\r\nfrom flask import current_app\r\n\r\n\r\ndef create_user(name: str, password: str):\r\n    with psycopg.connect(\"host=localhost user=postgres password=Prktsql1. port=5432 dbname=postgres\") as conn:\r\n        with conn.cursor() as cur:\r\n            cur.execute(\r\n                \"INSERT INTO users (name, password) VALUES (%s, %s)\",\r\n                (name, password)\r\n            )\r\n            conn.commit()\r\n\r\n\r\ndef check_user(name: str, password: str):\r\n    with psycopg.connect(\"host=localhost user=postgres password=Prktsql1. port=5432 dbname=postgres\") as conn:\r\n        with conn.cursor() as cur:\r\n            cur.execute(\r\n                \"SELECT id, password FROM users WHERE name = %s\",\r\n                (name,)\r\n            )\r\n            user_data = cur.fetchone()\r\n            if user_data and password == user_data[1]:\r\n                return user_data[0]\r\n            return None\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/database/user.py b/database/user.py
--- a/database/user.py	(revision 0e593b46ba452056e56fe87265aa3ec40e4dfe34)
+++ b/database/user.py	(date 1731351900788)
@@ -3,7 +3,7 @@
 
 
 def create_user(name: str, password: str):
-    with psycopg.connect("host=localhost user=postgres password=Prktsql1. port=5432 dbname=postgres") as conn:
+    with psycopg.connect("host=localhost user=postgres password=progre port=5432 dbname=postgres") as conn:
         with conn.cursor() as cur:
             cur.execute(
                 "INSERT INTO users (name, password) VALUES (%s, %s)",
@@ -13,7 +13,7 @@
 
 
 def check_user(name: str, password: str):
-    with psycopg.connect("host=localhost user=postgres password=Prktsql1. port=5432 dbname=postgres") as conn:
+    with psycopg.connect("host=localhost user=postgres password=progre port=5432 dbname=postgres") as conn:
         with conn.cursor() as cur:
             cur.execute(
                 "SELECT id, password FROM users WHERE name = %s",
Index: config.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>class Config:\r\n    # Enter your OpenWeather API key here\r\n    OPEN_WEATHER_API_KEY = \"<your_secret_key>\"\r\n    # Connection string used for postgres database access.'your_secret_key'\r\n    POSTGRES_CONNECTION_STRING = \"host=localhost user=postgres password=Prktsql1. port=5432 dbname=postgres\"
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/config.py b/config.py
--- a/config.py	(revision 0e593b46ba452056e56fe87265aa3ec40e4dfe34)
+++ b/config.py	(date 1731351507335)
@@ -2,4 +2,4 @@
     # Enter your OpenWeather API key here
     OPEN_WEATHER_API_KEY = "<your_secret_key>"
     # Connection string used for postgres database access.'your_secret_key'
-    POSTGRES_CONNECTION_STRING = "host=localhost user=postgres password=Prktsql1. port=5432 dbname=postgres"
\ No newline at end of file
+    POSTGRES_CONNECTION_STRING = "host=localhost user=postgres password=postgre port=5432 dbname=postgres"
\ No newline at end of file
